rpc/glfs-operations.c

```diff
@@ -8,7 +8,7 @@
   cases as published by the Free Software Foundation.
 */
 
-
+# include <../config.h>
```

Comment:
this doesn't look good! should be #include "config.h" or rather we should add '-include config.h' in CFLAGS in configure itself!